build.sbt

```diff
@@ -30,7 +30,7 @@ libraryDependencies ++= Seq(
   "commons-io" % "commons-io" % "2.4",
   "joda-time" % "joda-time" % "2.9.1",
   "com.typesafe" % "config" % "1.3.0",
-  "org.scalatest" % "scalatest_2.11" % "2.2.6" % "test"
+  "org.scalatest" % "scalatest_2.12" % "3.0.5" % "test"
```

Comment:
if you use `%%` instead of `%` it uses the scala version you are compiling with, if that helps